.github/workflows/test_onnxruntime_gpu.yml

```diff
@@ -51,14 +51,12 @@ jobs:
     steps:
       - name: Checkout
```

Comment:
What about triggering the workflow both when a PR is merged to main and nightly(and workflow_dispatch of course), something like this
```
on:
  workflow_dispatch:
  pull_request:
    branches: 
      - main
    types: [closed]
  schedule:
    - cron: 0 7 * * * #every day at 7 am
```

Reply:
added!